From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Arnav Singh <me@arnavion.dev>
Date: Sun, 30 Nov 2025 10:53:26 -0800
Subject: arm64: dts: sun50i-a64-pinephone-1.2: Increase backlight
 lth-brightness to 12

DanctNIX has been carrying a patch to raise this using udev because
10 apparently still turns the screen completely off for some phones.
A user in the postmarketOS tracker also reported this on their 1.2b model.
10 does seem to be enough for my phone, but let's raise it to 12.

Ref: https://codeberg.org/DanctNIX/danctnix-packages/src/commit/92039e7e20cef4aaa2ce8a92ca69fb12a53e806e/pine64/device-pine64-pinephone/10-pinephone-brightness.rules
Ref: https://gitlab.postmarketos.org/postmarketOS/pmaports/-/merge_requests/7441

Signed-off-by: Arnav Singh <me@arnavion.dev>
---
 arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.2.dts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.2.dts b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.2.dts
index 111111111111..222222222222 100644
--- a/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.2.dts
+++ b/arch/arm64/boot/dts/allwinner/sun50i-a64-pinephone-1.2.dts
@@ -21,7 +21,7 @@ &axp803 {
 
 &backlight {
 	power-supply = <&reg_ldo_io0>;
-	lth-brightness = <10>;
+	lth-brightness = <12>;
 };
 
 &lis3mdl {
-- 
Armbian

